docs/CONTRIBUTING.md

```diff
@@ -1,113 +1,972 @@
+
+### API Documentation Standards
+
+All public APIs must include comprehensive KDoc:
+
+```kotlin
+/**
+ * Badge provides visual status indication following Spark Design System principles.
+ *
+ * This component implements semantic color intentions and consistent typography
+ * to communicate status, counts, or categories within the interface.
+ *
+ * @param text The text content to display within the badge
+ * @param modifier [Modifier] to be applied to the badge layout
+ * @param intent [BadgeIntent] defining the semantic color treatment
+ * @param size [BadgeStyle] defining the typography and padding scale
+ * 
+ * @sample BasicBadgeSample
+ * @sample BadgeIntentSample
+ */
+@Composable
+public fun Badge(/* parameters */) { /* implementation */ }
+```
+
+---
+
+## 🔄 Pull Request Process
+
+The best way to make an impact is by creating code submissions called pull requests. Pull requests should be solutions to GitHub issues.
+
+### Submitting a Pull Request
+
+1. **Create an issue first** - Make sure there's a GitHub issue for the change you're proposing
+2. **Fork the repository** - [Fork](https://github.com/leboncoin/spark-android/fork) the repo for your contributions
+3. **Create a feature branch** - Write code in your fork, on a branch if you plan to make multiple changes
+4. **Follow the template** - [Create a pull request](https://help.github.com/articles/creating-a-pull-request/) following our [pull request template](.github/pull_request_template.md)
+5. **Wait for review** - A **[leboncoin/android](https://github.com/orgs/leboncoin/teams/android)** team member will triage and review your pull request
+6. **Merge** - If we accept your pull request, the accepting member will merge it for you
+
+### Pull Request Requirements
+
+Before submitting, ensure your PR:
+
+- [ ] Addresses a specific GitHub issue
+- [ ] Follows the architectural standards
+- [ ] Includes comprehensive tests
+- [ ] Has complete documentation
+- [ ] Passes all automated checks
+
+### Code Review Guidelines
+
+All contributions follow established code review practices focusing on:
+
+- **API Design**: Consistency with existing patterns
+- **Visual Quality**: Pixel-perfect implementation of designs
+- **Accessibility**: Compliance with accessibility standards
+- **Testing**: Comprehensive coverage of functionality
+- **Documentation**: Clear and complete API documentation
+
+---
+
+## ✨ Code Quality
+
+### Code Formatting
+
+Code style is enforced through Spotless with KtLint integration:
+
+```bash
+# Apply formatting automatically
+./gradlew spotlessApply
+
+# Verify formatting compliance
+./gradlew spotlessCheck
+```
+
+Alternatively, use the `✨ spotlessApply` run configuration.
+
+### Lint Checks
+
+The project includes custom lint rules that enforce Spark Design System patterns:
 
 ```bash
-# Install
-brew install git-lfs
-# Check
-git lfs install
+# Run all lint checks
+./gradlew lintRelease
+
+# Run lint with detailed output
+./gradlew lintRelease --info
```

Comment:
I'm not sure what is the point of explaining this Gradle option. This will not show more lint-related info.